contracts/ScopeGuard.sol

```diff
@@ -103,6 +117,12 @@ contract ScopeGuard is FactoryFriendly, BaseGuard {
         return (allowedTargets[target].scoped);
     }
 
+    /// @dev Returns bool to indicate if an address is scoped.
```

Comment:
wrong doc comment

```suggestion
    /// @dev Returns bool to indicate if sending empty data is allowed
```
```suggestion
    /// @dev Returns bool to indicate if an address is scoped.
```

Reply:
Will update to `@dev Returns bool to indicate if allowed to send to a target`

Reply:
> not related to this PR, but while looking at it I was wondering: Would it be useful to also support controlling whether sending ETH via value is allowed? (In case we'd ever add this, isSendAllowed would be a little ambiguous.)

I think that would be a solid improvement, but is out of scope for this PR.
Feel free to make an issue for it and someone can either build it during an upcoming cooldown or we can shape it into a project to work on in-cycle.